modules/checklist/assets/js/components/checklist-wrapper.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+import List from '@elementor/ui/List';
+import { steps } from '../data/steps';
+import CheckListItem from './checklist-item';
+
+function ChecklistWrapper() {
+	const [ expandedIndex, setExpandedIndex ] = useState( '0' );
+
+	return (
+		<List component="div" sx={ { py: 0 } }>
+			{
+				steps.map( ( step, index ) => {
+					return (
+						<CheckListItem step={ step } key={ index } setExpandedIndex={ setExpandedIndex } expandedIndex={ expandedIndex } />
```

Comment:
```suggestion
						<CheckListItem step={ step } index={ index } key={ index } setExpandedIndex={ setExpandedIndex } expandedIndex={ expandedIndex } />
```

Reply:
Why do we need `key` and `index`? @ronenelementor @elchugreeva

Reply:
Key is for the map, and index to use inside the item component, so when an item is clicked its index could be set or unset in the state